exist-versioning-release.md

```diff
@@ -147,11 +147,10 @@ You will require a system with:
                 <password>YOUR-PASSWORD</password>
             </server>
 
-            <!-- eXist-db BinTray -->
+            <!-- eXist-db Github Release -->
             <server>
-                <id>exist-bintray</id>
-                <username>YOUR-USERNAME</username>
-                <password>YOUR-PASSWORD</password>
+                <id>github</id>
+                <privateKey>[Github Personal access tokens]</privateKey>
```

Comment:
What about the GitHub username? I don't think the token by itself is enough

Reply:
a Github Userame is ont required but the <privateKey> is enough. Has been tested with several repos and works perfectly fine. Other way round, in case you have 2FactorAuth enabled (which we all have in the eXist-db github org) username / password will not work.